internal/service/iam/service_linked_role.go

```diff
@@ -196,7 +204,15 @@ func resourceServiceLinkedRoleUpdate(d *schema.ResourceData, meta interface{}) e
 	if d.HasChange("tags_all") {
 		o, n := d.GetChange("tags_all")
 
-		if err := roleUpdateTags(conn, roleName, o, n); err != nil {
+		err := roleUpdateTags(conn, roleName, o, n)
+
+		// If default tags only, log and continue. Otherwise, error.
+		if v, ok := d.GetOk("tags"); (!ok || len(v.(map[string]interface{})) == 0) && verify.CheckISOErrorTagsUnsupported(err) {
+			log.Printf("[WARN] failed adding tags after create for IAM Service Linked Role (%s): %s", d.Id(), err)
```

Comment:
This is in the update operation

```suggestion
			log.Printf("[WARN] failed updating tags for IAM Service Linked Role (%s): %s", d.Id(), err)
```